nni/compression/experiment/experiment.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from __future__ import annotations
+import logging
+from pathlib import Path
+from typing import Any, Callable, List
+
+import torch
+from torch.nn import Module
+from torch.optim import Optimizer
+
+from nni.compression.experiment.config import generate_compression_search_space
```

Comment:
Are they config utils, or experiment utils?

Reply:
compression search space generation relay on config, and search space is a part of experiment config right now, so put this function in `.config.utils`